### DISCUSSION TITLE
Conflict with other IOS package

### DISCUSSION THREAD

**Discussion by helgehoi** [HaishinKit/HaishinKit.swift#1403] [Q&A]
Dear shogo4405,

First of all, thank you so much for this package. 

I have encountered a problem that I hope you can help with. When installing HaishinKit in XCode for my ios app together with the package for C2PA that can be found here (see link) I get a build error.
https://gitlab.com/guardianproject/proofmode/simple-c2pa

Is there anything you could do to prevent this from happening?

Many thanks in advance,
Helge

The error is this one:
```
Multiple commands produce '/Users/helge/Library/Developer/Xcode/DerivedData/CR_Reporter-cmkeklqrrbbrjoctfstvjhhjzyec/Build/Products/Debug-iphonesimulator/include/module.modulemap'

duplicate output file '/Users/helge/Library/Developer/Xcode/DerivedData/CR_Reporter-cmkeklqrrbbrjoctfstvjhhjzyec/Build/Products/Debug-iphonesimulator/include/module.modulemap' on task: ProcessXCFramework /Users/helge/Library/Developer/Xcode/DerivedData/CR_Reporter-cmkeklqrrbbrjoctfstvjhhjzyec/SourcePackages/checkouts/HaishinKit.swift/Vendor/SRT/libsrt.xcframework /Users/helge/Library/Developer/Xcode/DerivedData/CR_Reporter-cmkeklqrrbbrjoctfstvjhhjzyec/Build/Products/Debug-iphonesimulator/libsrt.a ios simulator
```

**Comment by shogo4405** [comment]
I don't have the expertise to help with resolving build errors.

**Reply by helgehoi** [reply]
Thank you for the quick answer. It seems to me from googling the case that this could be resolved if either of the packages was designed to NOT use the default module.modulemap file and give it another name. Without being any expert it seems that if you don't rename this in your library, then it won't be able to co-exist with other libraries that include a similar bin. 

I can't find any way users of your library can fix this ourselves. It seems to me that it needs to be fixed in the library itself. If you want to see what happens, please try to install the library I added above.

Sorry for bothering you,
Helge

**Reply by shogo4405** [reply]
I'm waiting your PR.

**Reply by helgehoi** [reply]
Thank you for the invitation. Unfortunately, this is way beyond my competence. Hopefully somebody else sees this and knows what to do.

**Reply by helgehoi** [reply]
I don't know if this gives any direction:
https://stackoverflow.com/questions/75762015/multiple-swift-packages-with-xcframework-binary-targets-result-in-processxcfram

**Comment by shogo4405** [comment]
fixed https://github.com/shogo4405/HaishinKit.swift/pull/1660
